packages/core-logger/src/logger.ts

```diff
@@ -60,7 +60,13 @@ export abstract class AbstractLogger {
      * @param  {Number} figures
      * @return {void}
      */
-    public abstract printTracker(title: string, current: number, max: number, postTitle: string, figures: number): void;
+    public abstract printTracker(
```

Comment:
Did `tslint` format it like that?

Reply:
Yes :(

Reply:
Odd, never yet complained to me about that line.